README.md

```diff
@@ -1,64 +1,92 @@
-# **OSSF Endusers**
+# OSSF End Users
 
-[Brief description of the initiative]
+Anyone is welcome to join our open discussions.
 
+## Quick Start
 
-## 
-**Motivation**
+### Meeting times
 
-[Background / use cases of the problem to be solved]
+Every 2 weeks, Thursday 10am EST/3pm UTC. The meeting invite with zoom details is available on the [public OSSF calendar](https://calendar.google.com/calendar?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
 
+### Meeting notes
 
-## 
-**Objective**
+Meeting notes are [maintained in a Google Doc](https://docs.google.com/document/d/1KQalBRzfRBvsqh73JUYfp1KG-AJdXcv2Z8LTIFoQP8c/edit#). If attending please add your name, and if a returning attendee, please change the color of your name from gray to black.
 
-[What is to be achieved with this initiative]
+### Areas that need contributions
 
-[OKRs - OPTIONAL]
+- WG Documentation
+- Others TBD
 
+### Where to file issues
 
-## 
-**Scope**
+Issues can be reviewed and filed [here](https://github.com/ossf/wg-endusers/issues)
 
-[What is in and out of scope]
+## Objective
 
+The End User Working Group aims to ensure that the distinct and impactful voice of end users is heard in the development and delivery of the technical vision of OpenSSF.
 
-## 
-**Prior Work**
+## Vision
 
+The End Users Working Group (WG) represents the interests of public and private sector organizations that primarily consume open source rather than produce it.
 
+We strive to:
 
-*   List of prior and/or related projects
+- Ensure that the use cases for end user consumption of open-source software are understood and factored into OSSF programs.
+- Provide the resources required by end users to develop and implement more efficient and effective strategies, processes, tools, best practices and solutions that secure software supply-chains.
+- Provide a forum for learning from the experience and insights of peers.
+- Create an End Users Working Group, with representation from key private industry, public sectors, and multiple geographical regions.
+- Establish end user representation and active participation in OpenSSF working groups and leadership, both in the TAC and the Governing Board.
 
-# 
-**Get Involved**
+## Scope
 
-*   Official communications occur on the [ADD LINK TO YOUR WG MAILING LIST] (ex: https://lists.openssf.org/g/openssf-tac/topics).  \
-[Manage your subscriptions to Open SSF mailing lists](https://lists.openssf.org/g/main/subgroups).
-*   [Add Slack information if availabable]
+Based on the objective, mission, and goals above, we look to deliver specific delivereables:
 
-### 
-Quick Start
+- _Ensure that the use cases for end users consumption of open- source software is understood and factored into OSSF programmes._
```

Comment:
Fixed. I ended up removing the plural and matching it to the previous sentence that was the same. It should read a bit better since end "users'" is a bit of a "mouthful."

Apologies for all the minor typos. I "copy-pastad" and didn't give it the more thorough review it should have had. 

Again, thanks for the review! Let me know if you find anything else.